servers/arvr_server.cpp

```diff
@@ -72,20 +72,25 @@ void ARVRServer::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("tracker_added", PropertyInfo(Variant::STRING, "name"), PropertyInfo(Variant::INT, "type"), PropertyInfo(Variant::INT, "id")));
 	ADD_SIGNAL(MethodInfo("tracker_removed", PropertyInfo(Variant::STRING, "name"), PropertyInfo(Variant::INT, "type"), PropertyInfo(Variant::INT, "id")));
+
+	ADD_SIGNAL(MethodInfo("world_scale_changed", PropertyInfo(Variant::REAL, "scale")));
 };
 
 real_t ARVRServer::get_world_scale() const {
 	return world_scale;
 };
 
 void ARVRServer::set_world_scale(real_t p_world_scale) {
-	if (world_scale < 0.01) {
-		world_scale = 0.01;
-	} else if (world_scale > 1000.0) {
-		world_scale = 1000.0;
-	};
+	if (world_scale != p_world_scale) {
```

Comment:
It's a matter of preference, but I tend to use early returns in this case, to avoid having to indent all the world. So:
```
if (world_scale == p_world_scale) {
    return;
}

// now do stuff
```

Reply:
lol, I'm one of those people who was taught to try and make methods reach their end. Exiting early often results in having to duplicate freeing up stuff that you can otherwise have in one place at the end. 

But I'm fine either way.